addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -72,7 +74,8 @@ test("keep the `more` popover active when hovering it", async () => {
     await contains(".o-discuss-Call");
     await contains(".o-discuss-CallActionList");
     await click("[title='More']");
-    const enterFullScreenSelector = ".o-dropdown-item[title='Enter Full Screen']";
+    const enterFullScreenSelector =
+        ".o-discuss-CallActionList-dropdownItem[title='Enter Full Screen']";
```

Comment:
Ok for now, but just note that it's not good for having `title` when the item has the name of action already in text content. This displays the "title" popover that is shown after a delay and it's annoying.

In a PR I work to harmonize discuss actions, I made sure to clean that. I'll probably convert the call action list dropdown to use the same template and therefore remove the `title`, so no problem to keep it for now.